discoverer/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["database"]
 
 [dependencies]
 async-std = { version = "1.12.0", features = [ "attributes", "tokio1" ] }
-sea-schema = { version = "^0.9.3", default-features = false, features = ["sqlx-sqlite", "sqlx-mysql", "sqlx-postgres", "runtime-async-std-native-tls", "discovery", "writer" ] }
+sea-schema = { version = "^0.9.3", git = "https://github.com/SeaQL/sea-schema", default-features = false, features = ["sqlx-sqlite", "sqlx-mysql", "sqlx-postgres", "runtime-async-std-native-tls", "discovery", "writer" ] }
```

Comment:
Please don't merge it now. I need to publish SeaSchema 0.9.4 in a bit and bump the version here.

Reply:
SeaSchema 0.9.4 published